README.md

```diff
@@ -2,11 +2,40 @@
 
 This is a tool for taking screenshots of WMDE fundraising banners on wikipedia.org in different browsers and resolutions.
 
+## Creating Screenshots
+
 To run the tool, you need to set your SauceLabs credentials:
 
     export SAUCE_USERNAME=your_username
     export SAUCE_KEY=your_api_keys
 
 Then you can run the screenshot tool:
 
-    npm -r esm index.js
\ No newline at end of file
+    npm -r esm index.js
```

Comment:
Should we put these commands into the `package.json` and run them from there? Or add our default makefile too?

Reply:
The command will change again when we make it a proper CLI tool (see `TODO.md`) with parameters and arguments (taking cues from `metadata_summary`). Then, like the `metadata_summary` it will become a standalone command. Because the tool always needs at least the campaign parameter, I think `create_screenshots campaign_name` is nicer to type on the CLI than `make screenshot CAMPAIGN=campaign_name`

We could add a `Makefile` for ci, test and setup though.